SPECS/ucx/ucx.signatures.json

```diff
@@ -1,5 +1,5 @@
 {
  "Signatures": {
-  "ucx-1.11.0.tar.gz": "b7189b69fe0e16e3c03784ef674e45687a9c520750bd74a45125c460ede37647"
+  "ucx-1.11.0.tar.gz": "4b202087076bc1c98f9249144f0c277a8ea88ad4ca6f404f94baa9cb3aebda6d"
```

Comment:
this filename should change

Reply:
right, changed now

Reply:
Giving our make command line an option of `SRPM_FILE_SIGNATURE_HANDLING=update` updates the `*.signatures.json` file automatically .. 
FYI .. Just incase you might not be aware :)